docs/src/basicproperties.md

```diff
@@ -0,0 +1,27 @@
+# Accessing Graph Properties
+The following is an overview of functions for accessing graph properties. For functions that modify graphs, see [Making and Modifying Graphs](@ref). 
+
+## Whole-Graph Properties:
```

Comment:
"Graph Properties"? Whole Graph sounds like Whole Wheat to me.

Reply:
`struct BranGraph <: AbstractGraph`